src/pages/main_layout.cr

```diff
@@ -21,35 +21,109 @@ abstract class MainLayout
-      # content
+      div class: "flex flex-col md:flex-row md:bg-white mx-auto md:rounded-lg md:shadow md:-mt-12 md:px-5 md:py-12" do
+        content_block "Say goodbye to slow", <<-TEXT
+        A simple hello world response is returned in 0.1ms. Rendering
+        complex JSON usually takes only a few milliseconds.
+        TEXT
+
+        content_block "Batteries included", <<-TEXT
+        Authentication, asset management, CORS, database ORM, and more can
+        all be included when creating a new Lucky project.
+        TEXT
+
+        content_block "Fewer bugs. More joy", <<-TEXT
+        Instead of finding bugs in QA or in production, Lucky is designed to
+        catch as many bugs as possible at compile time.
+        TEXT
+      end
+    end
+  end
+
+  private def content_block(title, body)
+    div class: "w-full md:w-1/3 sm:px-0 md:px-8 my-5 md:my-0" do
+      h3 title, class: "mb-3 text-black font-bold text-base"
+      para body, class: "leading-loose text-sm text-grey-darker"
+    end
+  end
+
+  private def render_freeform_text
+    div class: "container mx-auto px-6 lg:px-32 markdown-content my-16" do
+      div class: "lg:px-16" do
+        raw CustomMarkdownRenderer.render_to_html <<-TEXT
+        ### Why is modern web development so complicated?
+
+        **There are more choices in tech than ever**, and with it comes more questions and more complexity.
+
+        Have you ever thought:
+
+        * "Why do I have to write **so much boilerplate**?"
+        * "How do **so many bugs make it to production**?"
+        * "Why is my app **so slow**!?"
+        * "Why do I have to choose between fast prototyping and long term maintainability?"
+
+        Every performance issue and every bug steals precious time from making your service better.
+
+        ### What if you could get something up quickly *and* maintain it for years to come?
+
+        Imagine this:
+
+        * Generators for quick code creation
+        * Built in authentication (Optional)
+        * Secure by default
+        * Built in Webpack config to get a SPA off the ground fast (Optional)
+        * Insanely fast execution so users aren’t left waiting
+        * Powerful type system that helps catch bugs before they ever hit production
+
+        If this sounds interesting, [let's get started](#{Guides::GettingStarted::Installing.path}).
+
+        ### Who shouldn't use Lucky?
+
+        Lucky is new software. If you need a stable set of methods and
+        lots of third party libraries, Lucky is probably not for you (yet!).
+
+        Still not sure? Talk to us in our
+        [chatroom](https://gitter.im/luckyframework/Lobby) and we'd be happy
+        to help you figure out if Lucky is a good fit for you.
+
+        If you're not quite ready to use Lucky in production, consider
+        [subscribing to our blog](#{Blog::Index.path}) or [following us
+        @luckyframework](https://twitter.com/luckyframework).
+        TEXT
```

Comment:
And this @edwardloveall @jwoertink

Reply:
Meh, I get it, but I think people will get it's new from the Version number still being so low. Maybe something like

```
### Still wondering if Lucky is a fit?

Talk to us in our 
[chatroom](https://gitter.im/luckyframework/Lobby) and we'd be happy
to help you figure out if Lucky is a good fit for you.

If Lucky is missing anything you would need for production use, please [file an issue], 
and consider [subscribing.....
```

It just sort of reads with a little negativity to me. We should be like "If Lucky isn't right for you, let us know why so we can fix it" type narrative (if that makes sense)

Reply:
I've always liked when people talk about why not to use something, but I get your point. Saying it more like "See something missing that you need? Let us know in the chatroom or create an issue" is probably a better way to put it. How did you feel about the other text above it?

And thanks for taking a look :)

Reply:
Yeah, we just want to make sure it sounds inclusive for everyone. I think the rest of the page is great. I love how you explain exactly the issue people think about when looking for a new framework.

Reply:
I think @jwoertink has a good point. I might phrase it like:

> Don't see a feature that you need? Still not sure Lucky is right for you? Lucky is in active development and we'd love to hear what features you need. Join us in our `chatroom` and keep up to date with our `newsletter` and `twitter`.

Reply:
Yup. That new change reads a lot more positive! 👍